src/secp256k1/src/util.h

```diff
@@ -216,6 +216,24 @@ static SECP256K1_INLINE void memczero(void *s, size_t len, int flag) {
     }
 }
 
+/** Semantics like memcmp. Variable-time.
+ *
+ * We use this to avoid possible compiler bugs with memcmp, e.g.
+ * https://gcc.gnu.org/bugzilla/show_bug.cgi?id=95189
```

Comment:
Wow, that's a really bad bug. Fortunately it will not have affected our main build before we switched to Clang, because we use -O1, which I insisted on precisely because of GCC's history of awful miscompilations at -O2 and above.